public/locales/en/signup.json

```
@@ -15,6 +14,32 @@
     "textWaitingList": "You have signed up for the {{name}} reserve place. If a place becomes available for you, you will receive a separate confirmation message to your e-mail about your registration.",
     "title": "Thank you for signing up!"
   },
+  "contactPerson": {
+    "cannotEditContactPersonNotification": {
+      "linkText": "Edit the signup group",
+      "text": "The signup group's contact person information cannot be edited from this page. Contact information can be edited from the editing page of the signup group.",
```

```suggestion
      "text": "The signup group's contact person information cannot be edited from this page. Contact information can be edited from the signup group's editing page.",
```
This same change was done in the linkedcomponents PR